ser_scripts/model.py

```diff
@@ -0,0 +1,43 @@
+# model.py
+
+import torch
+from torch import optim
+import torch.nn as nn
+import torch.nn.functional as F
+
+def model_load(learning_rate):
+
+    class Net(nn.Module):
```

Comment:
defining the class in a function is generally best avoided... it's valid but can bite you... especially when using library code like this and if you call the function (model_load here) more than once.